PICS/Patient Information/HeightKey.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  HeightKey.swift
+//  PICS
+//
+//  Created by Catherine Zhang on 1/30/24.
+//
+import SpeziAccount
+import SpeziViews
+import SpeziFoundation
+import SpeziValidation
+import SwiftUI
+
+/// The email address of a user.
+public struct HeightKey: AccountKey {
+    
+    public typealias Value = Int
+
+    public static let name: LocalizedStringResource = "Height"
+
+    public static let category: AccountKeyCategory = .personalDetails
+
+}
+
+
+extension AccountKeys {
+    public var height: HeightKey.Type {
+        HeightKey.self
+    }
+}
+
+
+extension AccountValues {
+    public var height: Int? {
+        storage[HeightKey.self]
+    }
+}
+
+
+// MARK: - UI
+extension HeightKey {
+    public struct DataDisplay: DataDisplayView {
+        public typealias Key = HeightKey
+        private let height: Int
+        public init(_ value: Int) {
+            self.height = value
+        }
+        public var body: some View {
+            HStack {
+                Text(HeightKey.name)
+                Spacer()
+                Text("\(height) cm")
+                    .foregroundColor(.secondary)
+            }
+            .accessibilityElement(children: .combine)
+        }
+    }
+}
+extension HeightKey {
+    public struct DataEntry: DataEntryView {
+        public typealias Key = HeightKey
+        @Binding private var height: Int
+        public var body: some View {
+            TextField("Height", value:$height, formatter: NumberFormatter())
```

Comment:
We possibly want to do something like 
```
HStack {
        Text(HeightKey.name)
        Spacer()
        TextField("Height", value:$height, formatter: NumberFormatter())
             .frame(width: 120) // set frame width to left align.
    }
```
To format the UI (and same for the weight key). Otherwise, users will not know what data they are inputting. See the screenshot below where we use the above code for the HeightKey while the WeightKey remains the same as in the PR (so we can only see the number input for weight but not what the number is for).
<img width="327" alt="image" src="https://github.com/CS342/2024-PICS/assets/32094663/3cc0af5b-60a6-43ae-900c-27f08671a292">

We might also consider how to only allow input for numbers but we can do so later if it takes a long time.